crates/rspack_plugin_split_chunks_new/src/plugin/module_group.rs

```diff
@@ -69,7 +69,7 @@ impl SplitChunksPlugin {
       cache_group_index: usize,
       cache_group: &'a CacheGroup,
       selected_chunks: Box<[&'a Chunk]>,
-      selected_chunks_key: ChunksKey,
+      selected_chunks_key: String,
```

Comment:
Why change the ChunksKey here, chunk key is useful to avoid unnecessary modules be merged into chunk

Reply:
@JSerFeng `ChunksKey` is the main performance problem here from the traces I took, construction a `BigUint` and then serializing that to a `String` was taking a significant time.